proto/streamlit/proto/Alert.proto

```diff
@@ -19,6 +19,7 @@ syntax = "proto3";
 message Alert {
   // Content to display.
   string body = 1;
+  string icon = 3;
```

Comment:
nit: not too important, but I think the proto definition would be nicer if the `format` field were moved up here with the other two fields. The `Format` enum declaration can stay where it is (things work correctly even though it's defined after it is used a field) or be moved above the fields.

Reply:
Updated on [2a689ec](https://github.com/streamlit/streamlit/pull/4964/commits/2a689ec5a0f421e458fd3ebc944cb2d90c1783d1)